blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/BluefloodEnumRollup.java

```diff
@@ -46,4 +58,24 @@ public boolean equals(Object obj) {
         return hashedEnumValues2Count.equals(other.hashedEnumValues2Count);
     }
 
+    public static BluefloodEnumRollup buildRollupFromEnumRollups(Points<BluefloodEnumRollup> input) throws IOException {
+        BluefloodEnumRollup enumRollup = new BluefloodEnumRollup();
+        for (Points.Point<BluefloodEnumRollup> point : input.getPoints().values()) {
+            BluefloodEnumRollup pointData = point.getData();
+            Map<Long, Long> incomingHashedEnums = pointData.getHashedEnumValuesWithCounts();
+            Map<Long, Long> currentHashedEnums = enumRollup.getHashedEnumValuesWithCounts();
+
+            for (Long hash : incomingHashedEnums.keySet()) {
+                long incomingCount = incomingHashedEnums.get(hash);
+                if (currentHashedEnums.containsKey(hash)) {
+                    long currentCount = currentHashedEnums.get(hash);
+                    incomingCount+=currentCount;
+                }
+                enumRollup.getHashedEnumValuesWithCounts().put(hash, incomingCount);
```

Comment:
since you've created "currentHashedEnums" as a synonym for "enumRollup.getHashedEnumValuesWithCounts()", couldn't you use it here as well?  In other words, instead of this line, wouldn't it be clearer to just do:
           currentHashedEnums .put(hash, incomingCount);